test/e2e/fixture/scylla/basic.scyllacluster.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     - name: us-east-1a
       members: 1
       storage:
-        capacity: 100Mi
+        capacity: 200Mi
```

Comment:
can you elaborate on why did this need an adjustment?

Reply:
i mentioned it in commit and PR description, commit log grows to 98MB when second node joins. I saw out of space error for 200Mi PVC, so I ended up leaving 300Mi.